geomstats/learning/pca.py

```diff
@@ -91,12 +91,21 @@ def _infer_dimension_(spectrum, n_samples, n_features):
 
 
 class TangentPCA(_BasePCA):
-    """Tangent Principal component analysis (tPCA).
+    r"""Tangent Principal component analysis (tPCA).
 
     Linear dimensionality reduction using
     Singular Value Decomposition of the
     Riemannian Log of the data at the tangent space
     of the Frechet mean.
+
+    Parameters
+    ----------
+    metric : RiemannianMetric
+        Riemannian metric.
+    n_components : int
+        Number of principal components.
+    point_type : str, {\'vector\', \'matrix\'}
+        Point type.
```

Comment:
point_type is optional and by default it is metric.default_point_type

Reply:
in fact it should be removed..

Reply:
Why?